### ISSUE TITLE
Deploy failed with error for Tab App: attempted import error: 'isSpanContextValid' is not exported from '@azure/core-tracing'.

### ISSUE THREAD

**Issue by SLdragon** [OfficeDev/TeamsFx#1836]
**Describe the bug**
When deploy frontend page, it will show errors as below:
```
[2021-08-05T03:23:53.042Z] [Info] - [Tab Frontend] Start to run command: "npm run build".
[2021-08-05T03:32:52.341Z] [Debug] - [Tab Frontend] 
> teamsfx-template-tab@0.1.0 build c:\Users\rentu\Desktop\CollaborationTestProject\RentuTestCollab\tabs
> react-scripts build

Creating an optimized production build...
Failed to compile.

./node_modules/@azure/core-rest-pipeline/dist-esm/src/policies/tracingPolicy.js
Attempted import error: 'isSpanContextValid' is not exported from '@azure/core-tracing'.
```

**To Reproduce**
1. Create project with Tab
2. Run provision command
3. Run deploy command

VSCode extension version: 2.3.1

**Comment by KristapsD** [comment]
Having the same issue, when trying to deploy. Any news regarding this?

**Comment by hund030** [comment]
@KristapsD Toolkit fails to resolve nested dependency conflict. As a workaround, you could add `"@azure/core-tracing": "1.0.0-preview.13"` to your dependency. We are working on a longer term fix in toolkit.

**Comment by KristapsD** [comment]
@hund030 Thanks! it worked.

**Comment by hund030** [comment]
@KristapsD This issue is fixed in teams toolkit@2.5.0. You can safely remove @azure/core-tracing from your pacakge.json now.
